### DISCUSSION TITLE
Does this software support GEDCOM 7?

### DISCUSSION THREAD

**Discussion by weismannweb** [fisharebest/webtrees#4600] [Q&A]
I want to import media, noteds-stories and other data.

**Comment by fisharebest** [comment]
It depends what you mean by "support"?

webtrees treats GEDCOM 7 tags just the same as custom tags from any other application.

It can display them, mostly understand them, load them, save them, etc.

But it exports files with a GEDCOM 5.5.1 header.

> I want to import media, noteds-stories and other data.

GEDCOM 5.5.1 supports these.  What application created your GEDCOM file?

**Comment by Norwegian-Sardines** [comment]
Weismann,

I’m not sure if fisharebest will see your update since it was an edit to you first entry.  I’m adding to this comment so a new notification gets sent!

what application generated this v7.0 GEDCOM file?  

As far as SNOTE vs NOTE as part of a fact entry webtrees for example: 

1 BIRT
2 DATE 01 JUL 1961
2 PLAC Sandringham, Norfolk, England
2 ADDR Park House
2 SNOTE @X243@
2 SOUR @S215@
3 PAGE https://en.wikipedia.org/wiki/Diana,_Princess_of_Wales

will except this data from your other program.  If after importing you have a specific issue let us know!
